<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OrganizeDialog</class>
 <widget class="QDialog" name="OrganizeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>582</width>
    <height>858</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Organize Files</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../data/icons.qrc">
    <normaloff>:/icons/64x64/strawberry.png</normaloff>:/icons/64x64/strawberry.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="layout_organizedialog">
   <item>
    <layout class="QFormLayout" name="layout_copying">
     <item row="0" column="0">
      <widget class="QLabel" name="label_destination">
       <property name="text">
        <string>Destination</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="destination"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_after_copying">
       <property name="text">
        <string>After copying...</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="aftercopying">
       <item>
        <property name="text">
         <string>Keep the original files</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Delete the original files</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="FreeSpaceBar" name="free_space" native="true"/>
   </item>
   <item>
    <widget class="QGroupBox" name="groupbox_naming">
     <property name="title">
      <string>Naming options</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="LineTextEdit" name="naming">
          <property name="toolTip">
           <string>&lt;p&gt;Tokens start with %, for example: %artist %album %title &lt;/p&gt;

&lt;p&gt;If you surround sections of text that contain a token with curly-braces, that section will be hidden if the token is empty.&lt;/p&gt;</string>
          </property>
          <property name="lineWrapMode">
           <enum>QTextEdit::NoWrap</enum>
          </property>
          <property name="acceptRichText">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="insert">
          <property name="text">
           <string>Insert...</string>
          </property>
          <property name="popupMode">
           <enum>QToolButton::InstantPopup</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="remove_problematic">
        <property name="text">
         <string>Remove problematic characters from filenames</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="remove_non_fat">
        <property name="text">
         <string>Restrict to characters allowed on FAT filesystems</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="remove_non_ascii">
        <property name="text">
         <string>Restrict characters to ASCII</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="allow_ascii_ext">
        <property name="text">
         <string>Allow extended ASCII characters</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="replace_spaces">
        <property name="text">
         <string>Replace spaces with underscores</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="overwrite">
        <property name="text">
         <string>Overwrite existing files</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="mark_as_listened">
        <property name="text">
         <string>Mark as listened</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="albumcover">
        <property name="text">
         <string>Copy album cover artwork</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupbox_preview">
     <property name="title">
      <string>Preview</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QStackedWidget" name="preview_stack">
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="preview_page">
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QListWidget" name="preview"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="loading_page">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <spacer name="spacer_preview_1">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>264</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="BusyIndicator" name="loading_indicator" native="true">
            <property name="text" stdset="0">
             <string>Loading...</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="spacer_preview_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>264</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="eject_after">
     <property name="text">
      <string>Safely remove the device after copying</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="button_box">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok|QDialogButtonBox::RestoreDefaults</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FreeSpaceBar</class>
   <extends>QWidget</extends>
   <header>widgets/freespacebar.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LineTextEdit</class>
   <extends>QTextEdit</extends>
   <header>widgets/linetextedit.h</header>
  </customwidget>
  <customwidget>
   <class>BusyIndicator</class>
   <extends>QWidget</extends>
   <header>widgets/busyindicator.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>destination</tabstop>
  <tabstop>aftercopying</tabstop>
  <tabstop>naming</tabstop>
  <tabstop>insert</tabstop>
  <tabstop>remove_non_fat</tabstop>
  <tabstop>remove_non_ascii</tabstop>
  <tabstop>allow_ascii_ext</tabstop>
  <tabstop>replace_spaces</tabstop>
  <tabstop>overwrite</tabstop>
  <tabstop>mark_as_listened</tabstop>
  <tabstop>albumcover</tabstop>
  <tabstop>eject_after</tabstop>
  <tabstop>preview</tabstop>
 </tabstops>
 <resources>
  <include location="../../data/data.qrc"/>
  <include location="../../data/icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>button_box</sender>
   <signal>accepted()</signal>
   <receiver>OrganizeDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>487</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_box</sender>
   <signal>rejected()</signal>
   <receiver>OrganizeDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>487</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
